tests/testthat/test-8-methSeg.r

```diff
@@ -25,6 +25,27 @@ test_that("check if methSeg returns error for methylBase objects" ,{
   expect_error(methSeg(methylBase.obj,diagnostic.plot = FALSE))
 })
 
+test_that("check if methSeg with cores > 1 is the same as cores=1" ,{
+  a=methSeg(methylRawList.obj[[3]],diagnostic.plot = FALSE)
+  b=methSeg(methylRawList.obj[[3]],diagnostic.plot = FALSE, cores=2)
+  expect_equal(a,b)
+})
+
+test_that("check if methSeg with cores > 1 is the same as cores=1 (non-tabix file)" ,{
+  a=methSeg(methylRawList.obj[[3]],diagnostic.plot = FALSE)
+  b=methSeg(methylRawList.obj[[3]],diagnostic.plot = FALSE, cores=2)
+  expect_equal(a,b)
+})
+
+methylRawDB.obj <- methRead(
```

Comment:
Is this really a tabix based object?
I think you have to set `save.db=TRUE`.

Reply:
ok!

Reply:
if I am correct it's `dbtype="tabix"`